crypto/hd/hdpath_test.go

```diff
@@ -185,102 +184,6 @@ func TestDeriveHDPathRange(t *testing.T) {
 	}
 }
 
-func ExampleStringifyPathParams() {
-	path := hd.NewParams(44, 0, 0, false, 0)
-	fmt.Println(path.String())
-	path = hd.NewParams(44, 33, 7, true, 9)
-	fmt.Println(path.String())
-	// Output:
-	// m/44'/0'/0'/0/0
-	// m/44'/33'/7'/1/9
-}
-
-func ExampleSomeBIP32TestVecs() {
-	seed := mnemonicToSeed("barrel original fuel morning among eternal " +
```

Comment:
these were causing a bunch of build errors. not because we changed anything, but just cause these tests haven't gotten hit in a long time.

they don't do anything, and we're not maintaining this branch after v4 so imo we're clear to just delete instead spending any cycles fixing